README

```diff
@@ -0,0 +1,163 @@
+Scientific Data Base
+--------------------
+
+# Grant Hutchison
+    Supplied the correct orientations for the major planets, their moons, and a
+    number of asteroids and also worked on these data files:
+        Solarsys.ssc, nearstars.stc, extrasolar.ssc, extrasolar.stc, earth_locs.ssc
+
+# Fridger Schrempp
+    Complete NGC/IC galaxy database + local group galaxies (galaxies.dsc)
+        Data base on globular clusters (globulars.dsc)
+
+        Data base on visual and spectroscopic binaries (visualbins.stc,
+        spectbins.stc)
+
+        World-capitals.ssc
+
+    Asterisms.dat
+
+# Andrew Tribick
+    Significant update of the stars.dat base based on Gaia DR2 data
+    (see: https://github.com/ajtribick/celestia-gaia-stardb);
+	licensed under CC-BY-SA 4.0.
```

Comment:
extra space

Reply:
Weird, in Notepad++ I don't see an extra space.

Reply:
because this is TAB actually. and your editor shows it as 4 spaces. use only spaces